testing/setup.txt

```diff
@@ -0,0 +1,103 @@
+1. Get a recent Ruby version installed (>= 3.0)
+
+   MacOS:
+	* Using MacPorts:
+	    sudo port install ruby31
+	    sudo port select ruby ruby31
+	    add to $PATH: $HOME/.gem/ruby/ruby/3.1.0/bin
+	    sudo gem update --system --verbose --no-document
+	    sudo gem update --verbose --no-document
+	    
+   Linux:
+	* Using rvm:
+	    (install curl and gnupg2 if not in the base installation)
+	    gpg2 --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+	    curl -sSL https://get.rvm.io | bash -s stable --ruby
+	    rvm install 3.1
+	    rvm use 3.1 --default
+	    gem update --system --verbose --no-document
+	    gem update --verbose --no-document
+	* Using apt (Ubuntu):
+	    sudo apt-get install ruby-full
+
+2. Install the Ruby Selenium interface
+
+   gem install selenium-webdriver --verbose --no-document --user-install
+
+3. Install the web drivers for browsers
+
+   The Selenium tools use the W3C webdriver interface to communicate
+   between the the test cases and a web browser. The set of web
+   drivers to install depends on the browsers to be tested against.
+
+   The web driver binary should be downloaded from the sources below
+   and installed in some directory in your $PATH.  (Recommended:
+   install in $HOME/bin)
+
+   * Firefox
+
+     The geckodriver binary can be installed from
+     https://github.com/mozilla/geckodriver/releases
+
+   * Chrome
+
+     The chromedriver binary can be installed from
+     https://chromedriver.chromium.org/downloads
+
+   * Safari
+
+     When running on MacOS (the only system that supports Safari), the
+     webdriver interface is included in Safari and nothing extra need
+     be installed
+   
+4. Install the test runner interface
```

Comment:
Is this the interface you demo'd during our call? Am I correct to assume that we can also start these tests from the command line?

Reply:
Yes, that's the interface I demoed. I'll add instructions on how to run from the command line -- it's not nearly as convenient.

Reply:
Instructions added.